### PR TITLE
Typo on web clients

### PR DESCRIPTION
**Pull request by martinboam**
Typos corrected.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
 
 ### Requirements for using media processors
 
-Media Processors are always in the media path for voice applications and for Web cleints (for exampe, Teams cleint in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
+Media Processors are always in the media path for voice applications and for Web clients (for example, Teams client in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Media Processors are always in the media path for voice applications and for Web clients (for example, Teams clients in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ For information about how to configure two trunks on the same SBC, see the docum
 
 Media bypass is supported with all Teams endpoints.
 
-Note for webcleints (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even of it started as a bypass call. This happens automatically and does not require any actions on the administrator. 
+Note for web clients (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even of it started as a bypass call. This happens automatically and does not require any actions on the administrator.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Note for web clients (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even if it started as a bypass call. This happens automatically and does not require any actions from the administrator. 
```

File: Teams/direct-routing-plan-media-bypass.md (hunk 1 above)
Comment (LolaJacobsen) [reply Teams/direct-routing-plan-media-bypass.md]: commit, then I'll merge.

File: Teams/direct-routing-plan-media-bypass.md (hunk 2 above)
Comment (LolaJacobsen) [reply Teams/direct-routing-plan-media-bypass.md]: Commit, then I'll merge

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**